sass/partials/_time_to_retirement.scss

```diff
@@ -139,15 +132,208 @@ $main_bubble_diameter: 250px;
+}
+
+#explain-text {
+    font-size: $base_font_size;
+    margin-left: 75px;
+    margin-top: 20px;
+}
+
+input[type="text"].form-control {
+    text-align: center;
+    @include box-shadow(#fff 0px 0px 0px);
+    height: 50px;
+    width: 100px;
+    font-size: $base_font_size * 2;
+    margin: auto auto;
+    display: inline-block;
+    @include border-radius(5px);
+
+    &.assets-field {
+        width: 150px;
+    }
+
+    &.income-field {
+        width: 120px;
+    }
+
+}
+
+.icon-answer {
+    text-align: center;
+    display: block;
+    font-size: 5em;
+    color: #fff;
+}
+
+.dollar-label {
+    position: absolute;
+    font-size: 1.2em;
+    display: inline-block;
+}
+
+.dollar-ouput-label-exp {
+    @extend .dollar-label;
+    left: -$expenses-field-height / 4;
+    top: $expenses-field-height;
+    font-size: 1.5em;
+}
+
+.dollar-ouput-label {
+    @extend .dollar-label;
+    left: $control-button-height / 2;
+    top: 43px;
+}
+
+.dollar-ouput-label-assets {
+    @extend .dollar-label;
+    @extend .dollar-ouput-label;
+    left: 10px;
+}
+
+#expenses-field {
+    height: $expenses-field-height;
+    width: 100%;
+    font-size: $base_font_size * 3;
+    display: inline-block;
+}
+
+.control-change-button {
+    display: block;
+    margin: auto auto;
+    height: $control-button-height;
+    width: $control-button-height * 2;
+    font-size: 1.5em;
+    padding: 0 0;
+    outline: none !important;
+    @include box-shadow(none !important);
+
+    &:focus {
+        background: none;
+    }
+}
+
+.control-increase-button {
+    @include border-radius(0px); // override Bootstrap .btn
+    @include border-top-radius($control-button-height);
+    border-bottom: 0px;
+}
+
+.control-decrease-button {
+    @include border-radius(0px); // override Bootstrap .btn
+    @include border-bottom-radius($control-button-height);
+    border-top: 0px;
+}
+
+.controls-row {
+    .control-change-button {
+        display: block;
+        margin: auto auto;
+        height: $control-button-height * (2/3);
+        width: $control-button-height * (4/3);
+        font-size: 0.8em;
+    }
+    .control-increase-button {
+        @include border-radius(0px); // override Bootstrap .btn
+        @include border-top-radius($control-button-height * (3/2));
+        border-bottom: 0px;
+    }
+
+    .control-decrease-button {
+        @include border-radius(0px); // override Bootstrap .btn
+        @include border-bottom-radius($control-button-height * (3/2));
+        border-top: 0px;
+    }
+}
+
+#assumptions-border-top {
+    border-top: 1px solid rgba(0,0,0,0.2);
+    position: absolute;
+    left: 15%;
+    right: 15%;
+}
+
+#assumptions-pencil {
+    display: inline-block;
+    margin-left: 10px;
+    color: $dark-green;
+
+    &:hover {
+        cursor: pointer;
+    }
+}
+
+.assumptions-section {
+    margin-top: 20px;
+    position: relative;
+}
+
+.assumptions-title-wrapper {
+    margin-top: 20px;
+}
+
+.assumptionseditable {

```

Comment:
use a class name that's hyphenated between the different words so it's easier to read